Chap_API_Reserved_Keys.tex

```diff
@@ -370,6 +370,8 @@ \subsection{Process realm attributes}
 Process identifier of the parent process of the specified process - typically
 used to identify the application process that caused the job containing the
 specified process to be spawned (e.g., the process that called \refapi{PMIx_Spawn}).
+This attribute is only provided for a process if it was created by a call to
```

Comment:
In most cases, launcher tools (e.g., mpiexec) will call PMIx_Spawn at the end of the launch operation chain (that may be delegated to a local daemon, e.g., prted), so it appears that in almost all cases this value will be set.

Maybe need to clarify/add an example of how to identify if the ID correspond to the launcher tool or some other type of process (e.g., application tool, client app process, etc).

Reply:
ASC Feb 15, 2022 we reviewed this issue. The general idea is to do exactly what this says (that is, tools like mpiexec will also set PMIX_PARENT_ID). If this is found to cause some compatibility difficulties with existing clients that misunderstood the spec, or relied on non-compliant behavior in prior prte versions, we'll reassess need for more attributes that clarify the case dichotomy.